### ISSUE TITLE
[BUG] itemList "DENY_LIST" only applies to players with operator status

### ISSUE THREAD

**Issue by Qinak** [snowgears/shopbugs#389]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Type of bug

Other unexpected behaviour

### Environment

```markdown
- Server Version: Paper 399 (MC: 1.19.3)
- Shop Version: 1.8.4.2
```

### Server startup log

https://gist.github.com/Qinak/99907d37d12d5a7fce6549323c0980bd

### Shop config files

https://gist.github.com/Qinak/3cc25929db0f306e6c86d24b52108d3a

### Bug description

itemList set do "DENY_LIST" only matter to players with operator status

### Steps to reproduce

Setup itemList to deny elytra
Create a shop without having operator status

### Expected behaviour

Players should not be able to create shop

### Actual behaviour

Only operator status players get canceld

**Comment by OstlerDev** [comment]
Fixed in upcoming version v1.9.1.2!

**Comment by OstlerDev** [comment]
@Qinak just got this fixed, please give it a test! I will try to take a look at your other issue soon.
